### ISSUE TITLE
Mistakes in README.md

### ISSUE THREAD

**Issue by andy812** [kvaps/kube-linstor#2]
Template kube-linstor chart, and apply it:

```
    cd helm

    helm template kube-linstor \
      --namespace linstor \
      --set controller.db.user=linstor \
      --set controller.db.password=hackme \
      --set controller.db.controller.db.connectionUrl=jdbc:postgresql://linstor-db-stolon-proxy/linstor \
      --set controller.nodeSelector.node-role\\.kubernetes\\.io/master= \
      --set controller.tolerations[0].effect=NoSchedule,controller.tolerations[0].key=node-role.kubernetes.io/master \
      --set satellite.tolerations[0].effect=NoSchedule,satellite.tolerations[0].key=node-role.kubernetes.io/master \
      --set csi.controller.nodeSelector.node-role\\.kubernetes\\.io/master= \
      --set csi.controller.tolerations[0].effect=NoSchedule,csi.controller.tolerations[0].key=node-role.kubernetes.io/master \
      --set csi.node.tolerations[0].effect=NoSchedule,csi.node.tolerations[0].key=node-role.kubernetes.io/master \
      > linstor.yaml

    kubectl create -f linstor-db.yaml -n linstor

```
1. Wrong filename: linstor.yaml instead of linstor-db.yaml
2. Usage of "-n linstor" is not correct, because of errors:
```
kubectl create -f linstor.yaml -n linstor
configmap/linstor-satellite created
daemonset.apps/linstor-satellite created
configmap/linstor-stunnel created
secret/linstor-stunnel created
service/linstor-stunnel created
statefulset.apps/linstor-controller created
secret/linstor-controller created
the namespace from the provided object "kube-system" does not match the namespace "linstor". You must pass '--namespace=kube-system' to perform this operation.
the namespace from the provided object "kube-system" does not match the namespace "linstor". You must pass '--namespace=kube-system' to perform this operation.
... etc.
```

**Comment by kvaps** [comment]
Hi, thanks for the report!

I've updated helm-chart and README.md, please pull new changes and regenerate manifests using new chart, now it should include exact namespace parameter into output yaml file.

Cheers

**Comment by andy812** [comment]
@kvaps спасибо!)
I found one more issue in helm/kube-linstor/templates/linstor-controller.yaml. 
If stunnel.enabled=false, you should set controller.expose=true, otherwise csi can't call controller by service name. Maybe it's not an issue, just a note in Readme.

**Comment by kvaps** [comment]
There is probably need more automatic, eg `restApiBindAddress` also should be set to `0.0.0.0` if stunnel disabled, I will think about it.

Otherhand LINBIT guys [promised](https://github.com/LINBIT/linstor-server/issues/67#issuecomment-522464253) to publish new docs soon with SSL configuration, so hopefully we will not need stunnel in the near future.

Thanks for testing!

**Comment by andy812** [comment]
Unfortunately, i couldn't run Linstor with stunnel.enabled=true. I use restApiBindAddress=0.0.0.0 and LINSTOR_IP=linstor-controller.linstor (csi can't resolve service name without namespace)

**Comment by kvaps** [comment]
If you running with `stunnel.enabled=true`, LINSTOR_IP should be set to localhost, you don't need to pass requests directly to linstor-controller.
If `stunnel.enabled=false`, you're right there is should be namespace. I'll check this out.

**Comment by andy812** [comment]
If i running with stunnel.enable=true, i get error: "Get http://localhost:3370/v1/resource-definitions: read tcp 127.0.0.1:33272->127.0.0.1:3370: read: connection reset by peer"

**Comment by kvaps** [comment]
Please check stunnel logs:
```
kubectl logs -n kube-system linstor-csi-controller-0 stunnel
kubectl logs -n linstor linstor-controller-0 stunnel
```

**Comment by andy812** [comment]
> 
> 
> Please check stunnel logs:
> 
> ```
> kubectl logs -n kube-system linstor-csi-controller-0 stunnel
> kubectl logs -n linstor linstor-controller-0 stunnel
> ```
```
# kubectl logs -n linstor linstor-controller-0 stunnel
2019.08.20 08:06:24 LOG4[ui]: Insecure file permissions on /etc/stunnel/psk.txt
2019.08.20 08:06:43 LOG3[0]: SSL_accept: ssl/statem/extensions.c:1612: error:141FA0FD:SSL routines:tls_psk_do_binder:binder does not verify
```
```
# kubectl logs -n kube-system linstor-csi-controller-0 stunnel
2019.08.20 08:06:09 LOG4[ui]: Insecure file permissions on /etc/stunnel/psk.txt
2019.08.20 08:06:43 LOG3[0]: SSL_connect: ssl/record/rec_layer_s3.c:1536: error:14094417:SSL routines:ssl3_read_bytes:sslv3 alert illegal parameter
```

**Comment by kvaps** [comment]
Well, configuration is correct, I use the same in production, try to do some diagnostic steps:

1. Make sure that `linstor-controller` is running and working, try to exec into pod and run `linstor n l` for example.
2. Check the network connectivity between the nodes/namespaces, have you some network politics enforced?
3. Try to increase stunnel verbosity by editing `debug` value in `linstor-stunnel` configmap on for server and client on both namespaces: `linstor` and `kube-system`, of course don't forget to recreate pods after edits for reload configuration.
4. Try to exec into `linstor-csi-controller-0` pod and install curl there, then repeat command from the output:
```
curl -X GET http://localhost:3370/v1/resource-definitions
```
Hopefully you will find what's wrong. Good luck with that! :)

**Comment by andy812** [comment]
Ok, i'll try, thank you! What is your k8s version?

**Comment by kvaps** [comment]
My one is `v1.15.1`

**Comment by andy812** [comment]
```
helm template kube-linstor \
  --namespace linstor \
  --set controller.db.user=linstor \
  --set controller.db.password=hackme \
  --set controller.db.controller.db.connectionUrl=jdbc:postgresql://linstor-db-stolon-proxy/linstor \
  --set controller.nodeSelector.node-role\\.kubernetes\\.io/master= \
  --set controller.tolerations[0].effect=NoSchedule,controller.tolerations[0].key=node-role.kubernetes.io/master \
  --set satellite.tolerations[0].effect=NoSchedule,satellite.tolerations[0].key=node-role.kubernetes.io/master \
  --set csi.controller.nodeSelector.node-role\\.kubernetes\\.io/master= \
  --set csi.controller.tolerations[0].effect=NoSchedule,csi.controller.tolerations[0].key=node-role.kubernetes.io/master \
  --set csi.node.tolerations[0].effect=NoSchedule,csi.node.tolerations[0].key=node-role.kubernetes.io/master \
  > linstor.yaml
```
Please, fix `--set controller.db.controller.db.connectionUrl`, it should be `controller.db.connectionUrl`

**Comment by kvaps** [comment]
Fixed in caf927a, was problem in that?
